ocaml/tests/test_host.ml

```diff
@@ -23,6 +23,7 @@ let add_host __context name =
        ~external_auth_service_name:"" ~external_auth_configuration:[]
        ~license_params:[] ~edition:"" ~license_server:[]
        ~local_cache_sr:Ref.null ~chipset_info:[] ~ssl_legacy:false
+       ~last_software_update:Xapi_stdext_date.Date.never
```

Comment:
```suggestion
       ~last_software_update:Xapi_stdext_date.Date.epoch
```

Reply:
Aha, there's 36 more of those in the existing code...

Reply:
I'll change them all when I specsave the libraries into xapi

there are a few deprecated bindings in the libirary: https://ocaml.org/p/xapi-stdext-date/latest/doc/Xapi_stdext_date/Date/index.html#val-never